ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.11.yaml

```diff
@@ -55,6 +55,42 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: aws-ipi-baselinecaps-none-f14
+  cron: 13 11 7,22 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      BASELINE_CAPABILITY_SET: None
+      LOKI_ENABLED: "false"
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.13 and @aws-ipi and not @fips
+        and not @storage and not @console'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.13 and @aws-ipi and not @fips
+        and not @storage and not @console'
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
+    pre:
+    - chain: ipi-conf-aws
+    - chain: ipi-install
+    - ref: enable-qe-catalogsource
```

Comment:
I guess we can directly call `chain: cucushift-installer-rehearse-aws-ipi-provision` here?

Reply:
yeah, agree. We can even use the workflow directly.